src/integrationtests/java/com/aws/greengrass/integrationtests/BridgeTest.java

```diff
@@ -90,7 +92,7 @@ void GIVEN_mqtt3_and_mapping_between_local_and_iotcore_WHEN_local_message_receiv
                         .contentType("contentType")
                         .build());
 
-        assertTrue(messageReceived.await(10L, TimeUnit.SECONDS));
+        assertTrue(messageReceived.await(AWAIT_TIMEOUT_SECONDS, TimeUnit.SECONDS));
```

Comment:
this is a _local_ message. Why would it take 30 seconds? This seems quite bad

Reply:
yeah, im still digging a bit more into why.  actually probably not a timeout, we're getting a null message on the assertion below

Reply:
think it was a race, latest commit has a fix

Reply:
you're counting down _before_ setting the value.

Use our asyncAssertOnConsumer, or use a future to get both a timeout and a value.

Reply:
yeah, switched to asyncAssertOnConsumer 👍